modules/account/lib/Account.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Jelix\Authentication\Account;
+
+class Account
+{
+    protected $name;
+
+    protected $email;
+
+    protected $status;
+
+    protected $create_date;
+
+    protected $provider;
+
+    protected $account_id;
+
+    public static $properties = array('name', 'email', 'status', 'create_date', 'provider', 'account_id');
+
+    public static $editableProperties = array('email');
```

Comment:
the problem with these two properties above, is that it could not be modifiable easily if a developer redefine the DAO and the form (into var/overloads). remove them